README.md

```diff
@@ -241,6 +241,32 @@ This SDK exposes the following classes:
 - `TransactionsWidget`
 - `TrendsWidget`
 
+### Troubleshooting
+
+#### Post messages not working
+
+Check the following items if you have passed post message callbacks in your
+widget instantiation but they are not being triggered:
+
+- Ensure your SSO request is correctly configured for the Web SDK:
+    - `ui_message_version` should be set to `4`.
+    - `is_mobile_webview` should be set to `true`.
+- Ensure you the appropriate widget class for the `widget_type` used when
+  making the SSO request. For example, if you set `widget_type` to be
+  `connect_widget`, then you should use the `ConnectWidget` class expected by
+  the SDK.
+- Ensure you are serving your application's HTML file from a web server and not
+  loading it from your file system.
+- Check your application's [`Referrer-Policy`][referrer_policy]. Due to the
```

Comment:
Do we know which `Referrer-Policy` values do and do not work? I'm guessing any policy that sends us the Referrer header will work. So `same-origin` won't work, but most others will (maybe all others), including [`strict-origin`](https://www.w3.org/TR/referrer-policy/#referrer-policy-strict-origin) depending on how they set it up -- it'd need to test though. But do we have someone internally that knows?